src/main/kotlin/org/wa/rceditor/domain/MainUseCase.kt

```diff
@@ -0,0 +1,19 @@
+package org.wa.rceditor.domain
+
+import io.reactivex.Completable
+import io.reactivex.Maybe
+import org.wycliffeassociates.resourcecontainer.ResourceContainer
+
+class MainUseCase(private var RCManager: RCManager) {
```

Comment:
Better to split this use case into smaller parts and/or give a more descriptive name. `AccessResourceContainer`? Or maybe `CreateResourceContainer` and `ReadWriteResourceContainer`?

Reply:
@mbr4477 I didn't understand why to split them. Can you please explain this to me. As for me it's just 3 methods. I don't understand why to split this into 2-3 classes.